src/v/datalake/tests/CMakeLists.txt

```diff
@@ -40,4 +40,20 @@ rp_test(
     v::iceberg_test_utils
   LABELS storage
   ARGS "-- -c 1"
+)
+
+rp_test(
+  GTEST
+  BINARY_NAME parquet_writer_test
+  SOURCES parquet_writer_test.cc
+  LIBRARIES
+    v::application
+    v::features
+    v::gtest_main
+    v::kafka_test_utils
+    v::datalake
+    v::model_test_utils
+    v::iceberg_test_utils
+  LABELS storage
```

Comment:
nit: probably only need gtest, datalake, iceberg test utils? and the labels in this file are off